apps/studio/src/schemas/folder.ts

```diff
@@ -1,22 +1,25 @@
 import { z } from "zod"
 
+import { generateBasePermalinkSchema } from "./common"
+
 export const MAX_FOLDER_TITLE_LENGTH = 100
 export const MAX_FOLDER_PERMALINK_LENGTH = 200
 export const MAX_FOLDER_DESCRIPTION_LENGTH = 300
 
+const permalinkSchema = generateBasePermalinkSchema("folder")
+  .min(1, { message: "Enter a URL for this folder" })
+  .max(MAX_FOLDER_PERMALINK_LENGTH, {
+    message: `Folder URL should be shorter than ${MAX_FOLDER_PERMALINK_LENGTH} characters.`,
+  })
+
 export const createFolderSchema = z.object({
   folderTitle: z
     .string()
     .min(1, { message: "Enter a title for this folder" })
     .max(MAX_FOLDER_TITLE_LENGTH, {
```

Comment:
Should traversal validation be added here too? See first step of VAPT poc

Reply:
lemme clarify this iwth them

Reply:
<img width="386" alt="image" src="https://github.com/user-attachments/assets/3e11eed1-312c-40ca-a7df-3667f3cf1cd2">
double checked, the title is irrelevant here, so it's ok!